docs/administration/authentication/microsoft-azure-ad.md

```diff
@@ -61,6 +61,45 @@ Restart the NetBox services so that the new configuration takes effect. This is
 sudo systemctl restart netbox
 ```
 
+## Group Assignment
+
+If you want NetBox to assign groups based on Azure AD groups, then some additonal configuration is needed. Enter the following configuration parameters in `configuration.py`, substituting your own values:
+
+```python
+SOCIAL_AUTH_AZUREAD_OAUTH2_RESOURCE = 'https://graph.microsoft.com/'
+SOCIAL_AUTH_PIPELINE = (
+    'social_core.pipeline.social_auth.social_details',
+    'social_core.pipeline.social_auth.social_uid',
+    'social_core.pipeline.social_auth.social_user',
+    'social_core.pipeline.user.get_username',
+    'social_core.pipeline.social_auth.associate_by_email',
+    'social_core.pipeline.user.create_user',
+    'social_core.pipeline.social_auth.associate_user',
+    'netbox.authentication.user_default_groups_handler',
+    'social_core.pipeline.social_auth.load_extra_data',
+    'social_core.pipeline.user.user_details',
+    'netbox.authentication.azuread_map_groups',
+)
+
+# Define special user types using groups. Exercise great caution when assigning superuser status.
+SOCIAL_AUTH_PIPELINE_CONFIG = {
+    'AZUREAD_USER_FLAGS_BY_GROUP': {
+        "is_staff": ['{AZURE_GROUP_ID}',],
+        "is_superuser": ['{AZURE_GROUP_ID}',]
```

Comment:
IMO we should remove the braces from the example just to avoid confusion with the containing Python syntax. It might also be helpful to show an example listing multiple groups.

```suggestion
        "is_staff": ['StaffGroup1', 'StaffGroup2'],
        "is_superuser": ['SuperuserGroup'],
```